index.html

```diff
@@ -67,7 +67,7 @@ <h3>Sreelakshmi Jayarajan</h3>
       </p>
       <h3>Vishnu Sanal T</h3>
       <p>
-        Member at Moonshot, and a B.Tech CSE (S4) student at GEC Palakkad. These days he's working on open-source projects including <a href="https://www.GitHub.com/TeamAmaze/AmazeFileManager">Amaze File Manager</a>, <a href="https://www.GitHub.com/VishnuSanal/DialogMusicPlayer">DialogMusicPlayer</a> & <a href="https://www.GitHub.com/VishnuSanal/QuotesStatusCreator">QuotesStatusCreator</a>. Previously, he was a contributor to <a href="https://www.cmu.edu/scs/robotutor/index.html">RoboTutor</a>, Carnegie Mellon University and have volunteered as a mentor for FOSS OpenHack '20.
+        Member at Moonshot, and a B.Tech CSE (S4) student at GEC Palakkad. These days he's working on open-source projects including Amaze File Manager, DialogMusicPlayer & QuotesStatusCreator. Previously, he was a contributor to RoboTutor, Carnegie Mellon University and have volunteered as a mentor for FOSS OpenHack '20.
```

Comment:
Maybe you can put CMU in brackets because right now it seems like you contributed to two different entities: RoboTutor and CMU. You can also put "Android dev./eng." instead of "contributor" if you want. Also feel free to mention any impressive stats/facts, like I mentioned at Discord.